tests/testSupport/userFactory.js

```diff
@@ -0,0 +1,23 @@
+import { Factory } from 'fishery';
+import { faker } from '@faker-js/faker';
+import { User } from '../../src/models/User.mjs';
+
+const UserFactory = Factory.define(({ onCreate, sequence, params }) => {
+  onCreate(async (user) => await User.query().insert(user));

```

Comment:
_acredito_ que essas keywords possam ser removidas, já que a ideia aqui é que o retorno de`UserFactory.create` seja uma promise (exemplo do [README](https://github.com/thoughtbot/fishery?tab=readme-ov-file#asynchronously-create-objects-with-your-factories))

```suggestion
  onCreate((user) => User.query().insert(user));

```